src/overrides.jl

```diff
@@ -1,2 +1,7 @@
-adjacency_matrix(g::StaticGraph) = SparseMatrixCSC{Bool,UInt32}(nv(g), nv(g), g.f_ind, g.f_vec, ones(Bool, ne(g)*2))
-adjacency_matrix(g::StaticDiGraph) = SparseMatrixCSC{Bool,UInt32}(nv(g), nv(g), g.f_ind, g.f_vec, ones(Bool, ne(g)))
+function adjacency_matrix(g::StaticGraph)
```

Comment:
Why not parameterize here?
```
adjacency_matrix(g::StaticGraph{I},T::Type) where I<:Integer where T<:Real = 
    SparseMatrixCSC{T,I}(nv(g), nv(g), g.f_ind, g.f_vec, ones(T, ne(g)*2))
```

Also, we should have this override match the signature for LG:
```
function adjacency_matrix(g::AbstractGraph, T::DataType=Int; dir::Symbol=:out)
```

Reply:
Maybe in another PR? This will require additional tests, etc

Reply:
the only thing done here in this PR is replace the too-long one-liner with a `function() ... end` block. This can stay and be built upon in a future fix of the content

Reply:
Oh. Sorry. I thought you were improving on #8.